spec/models/miq_schedule_worker/runner_spec.rb

```diff
@@ -324,57 +324,49 @@
 
             it "queues the right items" do
               scheduled_jobs = @schedule_worker.schedules_for_database_operations_role
+              expect(scheduled_jobs.size).to eq(3)
```

Comment:
I'm assuming here that this should always be so, but if this makes this test fragile will happily change to something more general